frontend-shared/styles/variables.scss

```diff
@@ -0,0 +1,2 @@
+// Other colors
+$color-focus-outline: #51a7e8;
```

Comment:
plucked from original variables.scss file. I think we will need a two copies of this file, ideally with no overlap.